Infoblox/ddi/tests/query_log_dns_1.json

```
@@ -0,0 +1,36 @@
+{
+  "input": {
+    "message": "FORMERR resolving 'test.testing.io/AAAA/IN': 192.168.1.136#53"
+  },
+  "expected": {
+    "message": "FORMERR resolving 'test.testing.io/AAAA/IN': 192.168.1.136#53",
+    "event": {
+      "action": "FORMERR"
+    },
+    "dns": {
+      "header_flags": [],
+      "question": {
+        "class": "IN",
+        "name": "test.testing.io",
+        "registered_domain": "testing.io",
+        "subdomain": "test",
+        "top_level_domain": "io",
+        "type": "AAAA"
+      },
+      "type": "query"
+    },
+    "related": {
+      "hosts": [
+        "test.testing.io"
+      ],
+      "ip": [
+        "192.168.1.136"
+      ]
+    },
+    "source": {
+      "address": "192.168.1.136",
+      "ip": "192.168.1.136",
+      "port": 53
+    }
```

I think it's more the destination of the dns packet
```suggestion
    "destination": {
      "address": "192.168.1.136",
      "ip": "192.168.1.136",
      "port": 53
    }
```